app/javascript/lib/imperial.js

```diff
@@ -1555,14 +1607,33 @@ export default class Imperial {
   }
 
   unitCount(nation) {
+    return this.armyCount(nation) + this.fleetCount(nation);
+  }
+
+  armyCount(nation) {
     let out = 0;
     for (const [, units] of this.units.get(nation)) {
       out += units.armies;
+    }
+    return out;
+  }
+
+  fleetCount(nation) {
+    let out = 0;
+    for (const [, units] of this.units.get(nation)) {
       out += units.fleets;
     }
     return out;
   }
 
+  overUnitLimit(nation, unitType, numberOfPlacements) {
+    if (unitType === "armies") {
```

Comment:
For uniformity, what do you think of using `"army"` instead of `"armies"`? It's a very very minor thing but I think generally elsewhere in the codebase it would be `"army"`.